views/website/libraries/24-PHP.json

```diff
@@ -84,6 +84,7 @@
         "rs256": true,
         "rs384": true,
         "rs512": true,
+        "eddsa": true,
```

Comment:
This should be on the object above, though :smile:

Reply:
God, sorry!